### ISSUE TITLE
suggestion: add option for delete bookmark at context menu 

### ISSUE THREAD

**Issue by jazei-AR** [skyjake/lagrange#289]
suggestion: add option for delete bookmark at context menu 
If i want to to delete a bookmark of my list of them I can not delete one, doing right click mouse on it in bookmark page
![Sin título](https://user-images.githubusercontent.com/72522918/120889512-e5da6a80-c5d3-11eb-8c44-91e33a6ef536.jpg)
Regards!!!!
josé from Ar

**Comment by skyjake** [comment]
I agree it would be convenient if the link context menu had a "Delete Bookmark" item in addition to the "Bookmark Link" one. This would work with links on any page.
